src/model/wallet.ts

```diff
@@ -887,7 +887,9 @@ export const createWallet = async (
           ? (walletResult as Wallet)
           : new Wallet(pkey);
       setTimeout(() => {
-        dispatch(setIsWalletLoading(null));
+        if (ios || !isImported) {
```

Comment:
!imported = new wallet - then we use this logic for dismissing the loading state

Reply:
Maybe it makes sense to add it as a comment in the code?